package.json

```diff
@@ -39,7 +39,7 @@
     "lib": "lib"
   },
   "dependencies": {
-    "@elastic/elasticsearch": "7.15.0",
+    "@elastic/elasticsearch": "https://github.com/elastic/elasticsearch-js/archive/refs/tags/v7.13.0.tar.gz",
```

Comment:
```suggestion
    "@elastic/elasticsearch": "7.13.0",
```

Reply:
The whole point of this PR is to stop using NPM version because it's upgraded by npm-check-update